worksheet_B/Mandelbrot.cpp

```diff
@@ -15,36 +15,78 @@ int main()
 	const double minX = -2, maxX = 1, minY = -1.5, maxY = 1.5;
 
 	// Generate the image
-	for (int pixelY = 0; pixelY < image.height(); pixelY++)
+	for (int pixelY = 0; pixelY < ImageHeight; pixelY++)
 	{
-		// TODO: Map the y coordinate into the range minY to maxY
-		//double y0 =
+		// Maps the y coordinate into the range minY to maxY
+		double y0 = ((pixelY / ImageWidth) * (maxY - minY) + minY);
 
-		for (int pixelX = 0; pixelX < image.width(); pixelX++)
+		for (int pixelX = 0; pixelX < ImageWidth; pixelX++)
 		{
-			// TODO: Map the x coordinate into the range minX to maxX
-			//double x0 =
+			// Maps the x coordinate into the range minX to maxX
+			double x0 = ((pixelX / ImageWidth) * (maxX - minX) + minX);
+
+			// The algorithm to colour a single pixel (x0, y0) of the Mandelbrot set fractal
+			// http://rijndael.ece.vt.edu/challenge/codesigndata/2012_assign.pdf [online at 30/01/2017]
+			double x = 0;
+			double y = 0;
+			int iteration = 0;
+			int MaxIteration = 200;
```

Comment:
Some of your variable names start with lowercase letters and some with uppercase -- try to be consistent